New option for Organization admin to reset authenticator app for a member (#5123)
umbraco/UmbracoDocs: umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md

5. Click on the cogwheel and select the **Enforced MFA Method** from the drop-down list for the member.

Once it has been enabled, the next time the member logs in, they will be enforced to set up the MFA that has been chosen.

> An administrator is also able to reset the authenticator app settings for a member in the organization.

Reviewer: ```suggestion
It is possible for an administrator to reset the authenticator app settings for members of the organization.
```
